tests/serve_request.rs

```diff
@@ -220,3 +221,23 @@ fn serves_requests_custom_index_notice(tmpdir: TempDir, port: u16) -> Result<(),
 
     Ok(())
 }
+
+#[rstest]
+fn serves_requests_with_path_prefix() -> Result<(), Error> {
+    let route = "foobar";
+    let server = server(&["--path-prefix", route]);
+    let client = Client::new();
```

Comment:
You can probably get away without this line by using `reqwest::blocking::get` directly below where you need it. This is what we do in other tests and would keep this consistent.